pyramid_oereb/lib/renderer/extract/templates/xml/view_service.xml

```diff
@@ -9,7 +9,13 @@
 <data:LegendAtWeb>${map.legend_at_web|u}</data:LegendAtWeb>
 %endif
 %if map.legends:
-%for legend_entry in map.legends:
+<%
+    def sorting_function(legend_entry):
```

Comment:
Create this function out of the template an use the same as in the extract JSON ...